daemon/daemon_unix.go

```diff
@@ -1756,10 +1756,6 @@ func (daemon *Daemon) setupSeccompProfile() error {
 	return nil
 }
 
-func (daemon *Daemon) useShimV2() bool {
-	return cgroups.IsCgroup2UnifiedMode()
-}
-
```

Comment:
Would it be cleaner to keep this function and change it to something like?

```go
func (daemon *Daemon) useShimV2() bool {
	return cgroups.IsCgroup2UnifiedMode() || !daemon.disableShimV2
}
```

Reply:
I feel that is more complicated, I prefer removing `useShimV2()` function and use the single `disableShimV2` variable.